examples/AdWords/v201607/CampaignManagement/AddCompleteCampaignsUsingBatchJob.php

```diff
@@ -112,9 +112,10 @@ function AddCompleteCampaignUsingBatchJobExample(AdWordsUser $user) {
         $batchJob->status);
 
     $pollAttempts++;
-    if ($batchJob->status !== 'ACTIVE' &&
-        $batchJob->status !== 'AWAITING_FILE' &&
-        $batchJob->status !== 'CANCELING') {
+    if (!in_array($batchJob->status, array(
```

Comment:
Thanks for calling out the use of `BatchJobStatus` constants.
But I don't see much benefits to build an array and use in_array here.
Could you please leave only the use of the constants?
In addition, could you please indent lines 116, 117, 118 to comply with our coding convention?

Thanks!